<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Density Based Scan Algorithm</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Epsilon Distance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="SpinBox" name="epsilonSpin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Minimum Neighbors</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SpinBox" name="minNeighborsSpin"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Minimum points per cluster</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SpinBox" name="minDensitySpin"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="TableWidget" name="clusterTable">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <column>
         <property name="text">
          <string>Cluster #</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mean XY</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Point Count</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>groupBox</tabstop>
  <tabstop>epsilonSpin</tabstop>
  <tabstop>minNeighborsSpin</tabstop>
  <tabstop>minDensitySpin</tabstop>
  <tabstop>clusterTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
